.github/workflows/ci_backend_go.yaml

```
@@ -0,0 +1,40 @@
+
+      - name: Build
+        run: make build
+
+      - name: Format
+        run: make fmt
+
+      - name: Lint
+        run: make lint
+
+      - name: Test
+        run: make test
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)